polus-montage-plugin/README.md

```diff
@@ -0,0 +1,45 @@
+# Montage
+
+This plugin generates a stitching vector that will montage images together. The inputs are an image collection, a file pattern, and a layout array that indicates how the montage should be laid out. Below is a description of how the file pattern should be formatted and how to set the layout parameters.
+
+For more information on WIPP, visit the [official WIPP page](https://isg.nist.gov/deepzoomweb/software/wipp).
+
+## Input Filename Pattern
+
+This plugin uses [filename patterns](https://github.com/USNISTGOV/MIST/wiki/User-Guide#input-parameters) similar to that of what MIST uses. In particular, defining a filename variable is surrounded by `{}`, and the variable name and number of spaces dedicated to the variable are denoted by repeated characters for the variable. For example, if all filenames follow the structure `filename_TTT.ome.tif`, where TTT is a 0 padded number indicating the timepoint the image was captured at, then the filename pattern would be `filename_{ttt}.ome.tif`.
+
+Filename patterns may include `x` and `y` grid positions for each image or a sequential position `p`, but not both. This differs from MIST in that `r` and `c` are used to indicate grid row and column rather than `y` and `x` respectively. This change was made to remain consistent with Bioformats dimension names and to permit use of `c` as a channel variable.
+
+In addition to the position variables (both `x` and `y`, or `p`), the only other variables that can be used are `z`, `c`, `t`, and `r`.
+
+## Montage Layout
+
+The montage layout is specified by an array of strings. Each element in the array specifies the variables from the filename pattern that are used in every subgrid, starting from the most granular subgrid to the largest grid. This means that if the filename pattern is `filename_{xxx}_{yyy}_{ttt}.ome.tif` and the input array is `x,y,t`, this means that the images with the same `y` and `t` values but different `x` values are placed next to each other in a grid of images. Then, each grid of images is placed in another grid `t` is constant but `y` changes. Finally, all subgrids are placed into a grid according to `t`.
```

Comment:
typo, "Then, each grid of images is placed in another grid where `t` is constant..."